TINY-7731: Fixed table resizing when the contents of a cell has caused it's size to change (#7177)
tinymce/tinymce: modules/snooker/CHANGELOG.md

## Unreleased

### Changed
- The `pixelSize` and `percentSize` functions in `TableSize` no longer require the initial width to be passed.

### Fixed

> - Resizing table cells caused incorrect widths where the content had caused the cell to grow.

Reviewer: ```suggestion
- Resizing table cells caused incorrect widths in cases where those cells had grown to fit extra content.
```

## Unreleased

### Changed

> - The `pixelSize` and `percentSize` functions in `TableSize` no longer require the initial width to be passed.

Author: ```suggestion
- The `pixelSize` and `percentSize` functions in `TableSize` no longer require the initial width to be provided.
```
